src/main/java/jpo/sdw/depositor/controller/KafkaConsumerFactory.java

```diff
@@ -18,6 +19,8 @@ public static KafkaConsumer<String, String> createConsumer(DepositorProperties d
 
       props.put("key.deserializer", "org.apache.kafka.common.serialization.StringDeserializer");
       props.put("value.deserializer", "org.apache.kafka.common.serialization.StringDeserializer");
+      props.put(ConsumerConfig.FETCH_MIN_BYTES_CONFIG, 100000);
```

Comment:
Let's add a comment above this to explain why we're adding it. From an outside perspective it could seem pretty random.

Reply:
Comments added explaining the min_bytes and max_wait settings